api/utils/settings.py

```diff
@@ -31,5 +31,7 @@ class Settings(BaseSettings):
     MAIL_PORT: int = config('MAIL_PORT')
     MAIL_SERVER: str = config('MAIL_SERVER')
 
+    # FACEBOOK_APP_ID: str = config("FACEBOOK_APP_ID")
+    # FACEBOOK_APP_SECRET: str = config("FACEBOOK_APP_SECRET")
```

Comment:
remove your comments

Reply:
this is not a comment in order for my endpoint to work those should be uncommented. I will leave that for the devops guy

Reply:
Can I resolve this now?